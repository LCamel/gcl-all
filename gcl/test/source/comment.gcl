do True ->
  -- comment
    skip
    skip
    skip
  -- hey
 | True ->
    skip
    skip
    skip
od