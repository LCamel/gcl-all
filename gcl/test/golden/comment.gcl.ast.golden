do True ->

    skip
    skip
    skip

 | True ->
    skip
    skip
    skip
od